pubspec.yaml

```diff
@@ -3,7 +3,7 @@
 # license that can be found in the LICENSE file.
 
 name: codemirror
-version: 0.5.0+5.43.0
+version: 0.5.0+5.44.0
```

Comment:
`0.5.1` - lets bump the patch version as well

Reply:
Done